packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -106,13 +138,52 @@ export default class Reachability {
     }
   }
 
+  /**
+   * Reachability results as an object in the format that backend expects
+   *
+   * @returns {any} reachability results that need to be sent to the backend
+   */
+  async getReachabilityResults(): Promise<ReachabilityResults> {
+    let results;
+
+    // these are the only props that backend needs in the reachability results:
+    const reachabilityResultsProps = ['udp', 'tcp', 'xtls'];
+
+    try {
+      // @ts-ignore
+      const resultsJson = await this.webex.boundedStorage.get(
+        REACHABILITY.namespace,
+        REACHABILITY.localStorageResult
+      );
+
+      results = JSON.parse(resultsJson);
+
+      // remove any internal SDK data from the results, leave only the stuff that backend needs
+      Object.values(results).forEach((result) => {
+        Object.keys(result).forEach((prop) => {
+          if (!reachabilityResultsProps.includes(prop)) {
+            delete result[prop];
+          }
+        });
+      });
```

Comment:
please do not use `delete` operator because it can force browser to de-optimise code, also it is mutate in place.

you can use `mapValues` and `pick` from lodash, like this:
```suggestion
  results = mapValues(results, (result) => pick(result, reachabilityResultsProps))
```

Reply:
done, thanks for the suggestion, it's much better now